lib/tests/test_local_working_copy.rs

```diff
@@ -826,8 +825,7 @@ fn test_checkout_discard() {
     assert!(!file1_path.to_fs_path_unchecked(&workspace_root).is_file());
     assert!(file2_path.to_fs_path_unchecked(&workspace_root).is_file());
     let reloaded_wc =
-        LocalWorkingCopy::load(store.clone(), workspace_root, state_path, &user_settings())
-            .unwrap();
+        LocalWorkingCopy::load(store.clone(), workspace_root, state_path, repo.settings()).unwrap();
```

Comment:
nit: should be squashed to the first commit

Reply:
Done.